05SOA/RestaurantReviews/WebAPI/appsettings.json

```diff
@@ -6,5 +6,8 @@
       "Microsoft.Hosting.Lifetime": "Information"
     }
   },
+  "ConnectionStrings": {
```

Comment:
put this in appsettings.Development.json and please don't commit my connection string